erdpy_network/resources.py

```diff
@@ -15,3 +15,67 @@ def to_dictionary(self) -> Dict[str, Any]:
 class EmptyAddress:
     def bech32(self) -> str:
         return ""
+
+
+class SmartContractResult:
```

Comment:
OK, can stay like this for the moment. In a future PR, we should merge (after further analysis) this definition with the exiting class definition in `contract_results.py`.